src/main/java/org/folio/circulation/domain/policy/Period.java

```diff
@@ -26,6 +26,8 @@
 import io.vertx.core.json.JsonObject;
 
 public class Period {
+  private static final Period EMPTY_PERIOD = minutes(0);
```

Comment:
What is an empty period?

Reply:
See [Period#isEmpty](https://github.com/folio-org/mod-circulation/pull/638/files#diff-1b34dfc17c074ed5694994fcaf5e0d6bR206).

It is a period with zero duration.

Reply:
> It is a period with zero duration.

Maybe we could rename this to `zeroDurationPeriod`?

Reply:
Sure, renamed to `zeroDurationPeriod`